R/derive_vars_cat.R

```diff
@@ -0,0 +1,256 @@
+#' Derive Categorization Variables Like `AVALCATy` and `AVALCAyN`
+#' @param dataset
+#' `r roxygen_param_dataset(expected_vars = c("by_vars", "definition"))`
+#' @param definition List of expressions created by `exprs()`.
+#' Must be in rectangular format and specified using the same syntax as when creating
+#' a `tibble` using the `tribble()` function.
+#' The `definition` object it will be converted to a `tibble` using the `tribble()` function.
+#'
+#' Must contain:
+#'  - the column `condition` which evaluates to a logic in `dataset`.
```

Comment:
This one has me confused.  What about?
```suggestion
#'  - the column `condition` which will be converted to a logical expression and used on the `dataset` input.
```